recipes/crccheck/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "crccheck" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/crccheck-{{ version }}.tar.gz
+  sha256: 45962231cab62b82d05160553eebd9b60ef3ae79dc39527caef52e27f979fa96
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - crccheck
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://sourceforge.net/projects/crccheck/
+  summary: Calculation library for CRCs and checksums
+  doc_url: https://pythonhosted.org/crccheck/
+  license: GPL-3.0-or-later
+  license_file: LICENSE
```

Comment:
```
""" Base class for CRC and checksum classes.

  License::

    Copyright (C) 2015-2020 by Martin Scharrer <martin@scharrer-online.de>

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.

"""
```